start.sh

```diff
@@ -1,115 +1,104 @@
 #!/usr/bin/env bash
 set -ex
 
-GROUP=staff
-USER_UID=1001
-export HOME="/home/$USER"
+USER="rstudio"
+USER_UID="1000"
```

Comment:
As per my discussion with @davidread with UIDs and GIDs, I realise we can't actually make this change right now. We'd have to do some more serious refactoring and re-init'ing of containers and chown'ing